app/src/main/java/com/simplecity/amp_library/ui/adapters/PagerAdapter.java

```diff
@@ -21,8 +22,22 @@
      *
      * @param fragmentManager The {@link FragmentManager} to use
      */
-    public PagerAdapter(FragmentManager fragmentManager) {
+    public PagerAdapter(FragmentManager fragmentManager, boolean refreshPager) {
```

Comment:
Instead of passing a boolean here, why not just call `pagerAdapter.refreshPager()` externally, when you want to refresh the pager?

Reply:
I'm not create a separate mehod because when go back from settings onCreateView() is ever called with setupViewPager() that call new PageAdapter(), so i insert the boolean in the construc method

Reply:
You'll see what I mean when I commit my changes, give me half an hour or so.